docs/tables.md

```diff
@@ -186,4 +186,58 @@ Reduced Scans
 | asn        | INTEGER | Autonomous system number, eg. `13335` |
 | as_name    | STRING  | Autonomous system long name, eg. `Cloudflare, Inc.` |
 | result     | STRING  | The source type, followed by the `: null` (meaning success) or error returned. eg. `HTTPS: null`, `HTTPS: Incorrect web response: status lines don't match` |
+| error      | STRING  | An error classification, explained below. eg `read/tcp.tcp.timed_out` |
```

Comment:
tcp.tcp is mistyped. And I think we want `read/timed_out`

Reply:
fixed